doc/source/jbnmr_examples/s6_seperate_inner_phase.rst

```diff
@@ -5,6 +5,6 @@ The data used in this example is available for
 
 Listing S6
 
-[:download:`seperate.py <../../../examples/jbnmr_examples/s6_seperate_inner_phase/seperate.py>`]
+[:download:`separate.py <../../../examples/jbnmr_examples/s6_seperate_inner_phase/separate.py>`]
```

Comment:
this will break the link

Reply:
Something had gone wrong while pushing my final changes. Don't know what, but it's fixed now.